### DISCUSSION TITLE
[Scam Alert] Goes Up Higher Token

### DISCUSSION THREAD

**Discussion by AminMemariani** [Webamoozcom/warning-list#60] [Announcements]
<div dir="rtl">

Project Name:

GUH (Goes Up Higher)
نام پروژه:

نماد پروژه:

GUH

اتفاقی که افتاده است:
Arbitary Token
یک توکن ساختگی فاقد پشتوانه فنی بدون مارکت کپ و 
فعال در صرافی پنکیک سواپ
نوع هشدار:

کلاهبرداری

نوع کلاهبرداری:

شید کوین
Pump & Dump

وضعیت کلاهبرداری:
در جریان

لینک ها:
https://guhtoken.org/
https://coinmarketcap.com/currencies/goes-up-higher/
https://t.me/goesuphigher
https://www.reddit.com/r/goesuphigher/

**Comment by saeedkousha** [comment]
با سلام
آیا کامیونیتی فارسی هم دارن؟

**Reply by AminMemariani** [reply]
سلام و احترام
به فارسی خیر ولی اخیرا یکی که از لهجه اش معلوم بود آفریقایی هست در کلاب هاوس داشت تبلیغش رو میکرد برای چندتا ایرانی
من گفتم این پروژه اسکم هست ولی من رو انداختن بیرون
فکر کردم بهتر باشه زودتر ریپورت بشه قبل از اینکه گندش دربیاد

**Reply by saeedkousha** [reply]
البته همین مطلبی که توی سایتشون گذاشتن کاملا مشخص میکنه که پروژه اسکم هستش
Increasing Peg Price
For 360 epochs, GUH will increase 5% every 4 hours starting from an initial peg price of $.001.

**Comment by Webamoozcom** [comment]
تشکر، منتشر شد، یکی دو تا اشکال سینتاکس داشت که ناگیز شدم دستی درستش کنم.
